packages/web3-eth-contract/src/contract.ts

```diff
@@ -233,6 +234,7 @@ export class Contract<Abi extends ContractAbi>
 	private readonly _overloadedMethodAbis: Map<string, AbiFunctionFragment[]>;
 	private _methods!: ContractMethodsInterface<Abi>;
 	private _events!: ContractEventsInterface<Abi>;
+	private readonly _dataInputFill?: 'data' | 'input' | 'both';
```

Comment:
also add in docs, thanks

Reply:
document with info for which RPCs this will be used? sending tx, eth_call, estimate gas all?